src/wp-includes/block-template-utils.php

```diff
@@ -507,6 +507,64 @@ function _remove_theme_attribute_from_template_part_block( &$block ) {
 	}
 }
 
+/**
+ * Gets the contents for the given template file path, using a transient cache.
```

Comment:
> using a transient cache.

Not sure we should be documenting the implementation here, especially since the use of the cache is conditional.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5463/commits/ed704e60f093a01ebbcd31868c49a5d18bf3bf3c